test/test-format.ts

```diff
@@ -18,15 +18,18 @@ import test from 'ava';
 import fs from 'fs';
 import * as path from 'path';
 
-import {Options} from '../src/cli';
+import {Logger, Options} from '../src/cli';
 import * as format from '../src/format';
 import {nop} from '../src/util';
 
 import {withFixtures} from './fixtures';
 
 // clang-format won't pass this code because of trailing spaces.
-const BAD_CODE = 'export const foo = [ 2 ];';
-const GOOD_CODE = 'export const foo = [2];';
+const BAD_CODE = '//🦄\nexport const foo = [ \"2\" ];';
```

Comment:
As we discussed in person, let's have the unicorn be in its own test.

Reply:
done